TestingHomework.Tests/DataPrep/ProductPrep.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using TestingHomework.Tests.Data;
+using TestingHomework_Discounts;
+
+namespace TestingHomework.Tests.DataPrep
+{
+    public class ProductPrep : TypePrepBase<Product, Data.ProductDTO>
+    {
+        public ProductPrep(ITestDataAccessor dataAccessor) : base(dataAccessor)
+        {
+           
+        }       
+
+        public IEnumerable<Product> CreateManyForProducts(int count, Product product)
```

Comment:
What was the intent of this prep method?
It appears to be creating 'count' copies of the same project object.